optuna/importance/_ped_anova/evaluator.py

```diff
@@ -44,15 +44,9 @@ def filter(self, trials: list[FrozenTrial]) -> list[FrozenTrial]:
         err_msg = "len(trials) must be larger than or equal to min_n_top_trials"
         assert min_n_top_trials <= loss_values.size, err_msg
 
-        def _quantile(v: np.ndarray, q: float) -> float:
-            cutoff_index = int(np.ceil(q * loss_values.size)) - 1
-            return float(np.partition(loss_values, cutoff_index)[cutoff_index])
-
         cutoff_val = max(
             np.partition(loss_values, min_n_top_trials - 1)[min_n_top_trials - 1],
-            # TODO(nabenabe0928): After dropping Python3.7, replace below with
-            # np.quantile(loss_values, self._quantile, method="inverted_cdf").
-            _quantile(loss_values, self._quantile),
+            np.quantile(loss_values, self._quantile, method="inverted_cdf"),
```

Comment:
[note] `method` argument is available after Numpy v1.22.0 release.
https://github.com/numpy/numpy/releases/tag/v1.22.0

Reply:
I did not provide sufficient information, sorry :pray:

Reply:
But, we can install `numpy<1.22.0` in Python 3.8, isn't it?

Reply:
We discussed internally and decided to continue the support for `numpy<1.22.0`. Let me update this PR and then make ready for review.